apps/openmw/mwgui/dialogue.cpp

```diff
@@ -455,6 +456,7 @@ namespace MWGui
             mKeywordSearch.seed(Misc::StringUtils::lowerCase(*it), intptr_t(t));
         }
         mTopicsList->adjustSize();
+        mTopicsList->scrollToFirstItem();
```

Comment:
We can't scrollToFirstItem here. We get an updateTopics() call every time the player selects a topic. That means the list would incorrectly scroll to the top every time a topic is clicked.